src/components/FeaturedCard/index.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from './styles.module.css';
+
+import { FeaturedCards } from '../../lib/types';
+import Link from 'next/link';
+
+type FeaturedCardsItemProps = {
+  card: FeaturedCards;
+};
+
+const FeaturedCard: React.FC<FeaturedCardsItemProps> = ({ card }) => {
+  return (
+    <div
+      className={`${styles.card} lg:shadow-2xl shadow-xl lg:m-10 md:m-18 sm:m-5 mt-10 px-5 py-10 flex justify-between `}
+    >
+      <div className="relative">
+        <div
```

Comment:
You can close this `div` without children like this: `<div />`. Don't need a closing tag.